bumble/gatt.py

```diff
@@ -271,6 +271,26 @@ def __str__(self):
         return f'Characteristic(handle=0x{self.handle:04X}, end=0x{self.end_group_handle:04X}, uuid={self.uuid}, properties={Characteristic.properties_as_string(self.properties)})'
 
 
+# -----------------------------------------------------------------------------
+class CharacteristicDeclaration(Attribute):
+    '''
+    See Vol 3, Part G - 3.3.1 CHARACTERISTIC DECLARATION
+    '''
+    def __init__(self, uuid, properties, value_handle):
```

Comment:
Since this declaration is for a `Characteristic`, would it make sense to pass a `Characteristic` object to this constructor, instead of `uuid`, `properties` and `value_handle` separately, and keep a reference as `self.characteristic`?

Reply:
great suggestion, will do